biz.aQute.resolve/src/biz/aQute/resolve/AbstractResolveContext.java

```diff
@@ -105,6 +105,8 @@
 	private Set<Resource>							blacklistedResources		= new HashSet<Resource>();
 	private int										level						= 0;
 	private Resource								framework;
+	private Boolean									collectAllMissingRequirements;
```

Comment:
But what "external agents" are interrogating this trinary state of the AbstractResolveContext? :-) This boolean just hold the value of a Bnd instruction which has a default value.